src/Components/Dashboard/Dashboard.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import './Dashboard.css';
+import locationIcon from '../../assets/location.svg';
+import pollutionIcon from '../../assets/pollution.png';
+import humidityIcon from '../../assets/humidity.png';
+import pressureIcon from '../../assets/pressure.png';
+import windIcon from '../../assets/wind.png';
+import emptyFavIcon from '../../assets/star-empty.png';
+
+
+function Dashboard(props) {
+
+//CREATE METHOD THAT HANDLES FAV ICON CLICK (SAVE TO FAV LOCATIONS/LOCAL STORAGE)
+//CREATE METHOD THAT HANDLES AQI NUMBER CLICK TO BRING YOU TO INFORMATION PAGE
+
+  const airQualityMessages = (aqi) => {
+    if (aqi <= 50) {
```

Comment:
This is a great dynamic info display - I wonder if there's a way we can utilize hooks for this